src/pt/tsukimangas/src/eu/kanade/tachiyomi/extension/pt/tsukimangas/TsukiMangas.kt

```diff
@@ -0,0 +1,252 @@
+package eu.kanade.tachiyomi.extension.pt.tsukimangas
+
+import eu.kanade.tachiyomi.extension.pt.tsukimangas.dto.ChapterListDto
+import eu.kanade.tachiyomi.extension.pt.tsukimangas.dto.CompleteMangaDto
+import eu.kanade.tachiyomi.extension.pt.tsukimangas.dto.MangaListDto
+import eu.kanade.tachiyomi.extension.pt.tsukimangas.dto.PageListDto
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.network.asObservableSuccess
+import eu.kanade.tachiyomi.network.interceptor.rateLimitHost
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.decodeFromStream
+import okhttp3.HttpUrl
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Interceptor
+import okhttp3.Request
+import okhttp3.Response
+import rx.Observable
+import uy.kohesive.injekt.injectLazy
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class TsukiMangas : HttpSource() {
+
+    override val name = "Tsuki Mangás"
+
+    override val baseUrl = "https://tsuki-mangas.com"
+
+    private val API_URL = baseUrl + API_PATH
+
+    override val lang = "pt-BR"
+
+    override val supportsLatest = true
+
+    override val client by lazy {
+        network.client.newBuilder()
+            .rateLimitHost(baseUrl.toHttpUrl(), 2)
+            .rateLimitHost(MAIN_CDN.toHttpUrl(), 1)
+            .rateLimitHost(SECONDARY_CDN.toHttpUrl(), 1)
+            .addInterceptor(::imageCdnSwapper)
+            .build()
+    }
+
+    override fun headersBuilder() = super.headersBuilder().add("Referer", "$baseUrl/")
+
+    private val json: Json by injectLazy()
+
+    // ============================== Popular ===============================
```

Comment:
not really important but perhaps dont place the text in the middle. maybe 1st one like the others or 2nd if really need the line
```suggestion
    // Popular
```
```suggestion
    // Popular =============================================================
```